src/hooks/RestyProvider.jsx

```diff
@@ -1,12 +1,29 @@
-import React , { createContext } from 'react';
+import React, { createContext, useState, useContext } from 'react';
 
 const RestyContext = createContext;
```

Comment:
make sure to invoke createContext here on line 5 otherwise you will get some errors!